config/parameters.yaml.dist

```diff
@@ -1,25 +1,17 @@
 parameters:
-    secret: !ThisTokenIsNotSoSecretChangeIt!
-    chameleon_system_core.debug.show_view_source_html_hints: true
-    chameleon_system_core.development_email: "the developer's mail address"
-
-    database_host: "localhost"
-    database_port: 3306
-    database_name: "shp_dmo61"
-    database_user: "shp_dmo61"
-    database_password: "insert-password"
+    chameleon_system_core.development_email: "the developer's email address"
```

Comment:
Is this still necessary / or newly used?

Reply:
It's unchanged and still required.